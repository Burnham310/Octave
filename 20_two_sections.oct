guitar = Section 
    c1=[1 3 7], c2=[2 4 ^1], c6=[6 ^1 ^5], c4=[4 6 ^3], c5=[5 7 ^2];
    tonic=A, octave=5, mode=major, tempo=8/8, instrument=0, volume=33; {
    [] c2 [] c2 [] c2 [] c2
    [] c5 [] c5 [] c5 [] c5

    [] c1 [] c1 [] c1 [] c1
    [] c1 c1 [] [] c1 c1 []
}
bass = Section 
    ;
    tonic=A, octave=4, mode=major, tempo=8/8, instrument=1, volume=50; {
    6'/4 7'/4 ^2'/4 #2'/4
    3 5 7'/4 ~7'/4 6'/4
    5'/4 1'/4 ~3 3 5'/4
    [] 1 5 [] [] 7 ^1 []
}

piano = Section 
    c1=[.7 2 3 5 6];
    tonic=A, octave=5, mode=major, tempo=8/8, instrument=2, volume=33; {
    [] ^1 4 2 []'/4 6 3
    4  5 4 [] 2 #2 [] 7
    1 3'/16 ~3'/16 2'/16 ~7'/16 []'/2 ~.7
    .7 c1 c1 [] [] [.7 2 3 5 6 ^#2] [.7 2 3 5 6 ^#2] []
}

main = [guitar bass piano]
