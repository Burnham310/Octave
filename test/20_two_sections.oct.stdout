[
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 66,
  "duration": 0.5,
  "gap": 0,
  "amp": 0.5,
  "instrument": "sine-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 71,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 74,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 81,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 81,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 68,
  "duration": 0.5,
  "gap": 0,
  "amp": 0.5,
  "instrument": "sine-midikeyboard"
}
,
{
  "content": 74,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 71,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 74,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 81,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 71,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 71,
  "duration": 0.5,
  "gap": 0,
  "amp": 0.5,
  "instrument": "sine-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.5,
  "gap": 0.25,
  "amp": 0,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 71,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 74,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 81,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 60,
  "duration": 0.5,
  "gap": 0,
  "amp": 0.5,
  "instrument": "sine-midikeyboard"
}
,
{
  "content": 78,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 71,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 74,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 81,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 73,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 61,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.5,
  "instrument": "sine-midikeyboard"
}
,
{
  "content": 74,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 76,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 80,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 83,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 64,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.5,
  "instrument": "sine-midikeyboard"
}
,
{
  "content": 76,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 68,
  "duration": 0.5,
  "gap": 0,
  "amp": 0.5,
  "instrument": "sine-midikeyboard"
}
,
{
  "content": 74,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 76,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 80,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 83,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 67,
  "duration": 0.5,
  "gap": 0,
  "amp": 0.5,
  "instrument": "sine-midikeyboard"
}
,
{
  "content": 71,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 76,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 80,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 83,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 72,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 66,
  "duration": 0.5,
  "gap": 0,
  "amp": 0.5,
  "instrument": "sine-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 76,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 80,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 83,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 80,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 64,
  "duration": 0.5,
  "gap": 0,
  "amp": 0.5,
  "instrument": "sine-midikeyboard"
}
,
{
  "content": 69,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 69,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 73,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 80,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 73,
  "duration": 0.125,
  "gap": 0.125,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 72,
  "duration": 0.125,
  "gap": 0.125,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 57,
  "duration": 0.5,
  "gap": 0,
  "amp": 0.5,
  "instrument": "sine-midikeyboard"
}
,
{
  "content": 71,
  "duration": 0.125,
  "gap": 0.125,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 79,
  "duration": 0.125,
  "gap": 0.125,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 69,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 73,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 80,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": -930,
  "duration": 1,
  "gap": 0.25,
  "amp": 0,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 60,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0.5,
  "instrument": "sine-midikeyboard"
}
,
{
  "content": 69,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 73,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 80,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 61,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0.5,
  "instrument": "sine-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 64,
  "duration": 0.5,
  "gap": 0.25,
  "amp": 0.5,
  "instrument": "sine-midikeyboard"
}
,
{
  "content": 69,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 73,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 80,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 67,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": -942,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "sine-midikeyboard"
}
,
{
  "content": 68,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 69,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 73,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 80,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 57,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.5,
  "instrument": "sine-midikeyboard"
}
,
{
  "content": 68,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 71,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 73,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 76,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 78,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 69,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 73,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 80,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 64,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.5,
  "instrument": "sine-midikeyboard"
}
,
{
  "content": 68,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 71,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 73,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 76,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 78,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": -942,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "sine-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": -942,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "sine-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 69,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 73,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 80,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 68,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.5,
  "instrument": "sine-midikeyboard"
}
,
{
  "content": 68,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 71,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 73,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 76,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 78,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 84,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 69,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 73,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 80,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "acoustic-guitar"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 69,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.5,
  "instrument": "sine-midikeyboard"
}
,
{
  "content": 68,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 71,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 73,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 76,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 78,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 84,
  "duration": 0.25,
  "gap": 0,
  "amp": 0.33000001311302185,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "content": -942,
  "duration": 0.25,
  "gap": 0,
  "amp": 0,
  "instrument": "sine-midikeyboard"
}
,
{
  "content": -930,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0,
  "instrument": "triangle-midikeyboard"
}
,
{
  "content": 0,
  "duration": 0,
  "gap": 0,
  "amp": 0,
  "instrument": "dummy"
}

]